publish/publish.proj

```diff
@@ -6,9 +6,22 @@
   <UsingTask TaskName="GetAzureBlobList" AssemblyFile="$(LocalBuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.Local.dll" />
   <UsingTask TaskName="UploadToAzure" AssemblyFile="$(LocalBuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.Local.dll" />
 
-  <!-- ToDo: Update versions repo -->
   <Target Name="Build"
-          DependsOnTargets="PublishToAzure;PublishDebFilesToDebianRepo;FinalizeBuildInAzure;PublishCoreHostPackagesToFeed" />
+          DependsOnTargets="PublishToAzure;PublishDebFilesToDebianRepo;" />
+  
+  <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
+  <Target Name="PublishFinalOutput"
+          DependsOnTargets="FinalizeBuildInAzure;PublishCoreHostPackagesToFeed;UpdatePublishedVersions" />
+          
+  <Target Name="ExcludeSymbolsPackagesFromPublishedVersions" BeforeTargets="UpdatePublishedVersions" >
+    <RemoveDir Directories="$(BinDir)/publishpackages" />  
+    <ItemGroup>
+      <PackagesToShip Include="$(BinDir)/ForPublishing/*.nupkg" Exclude="$(BinDir)/ForPublishing/*.symbols.nupkg" />
+      <PackagesToShip Remove="%(PackagesToShip.Identity)" Condition="$([System.String]::Copy('%(PackagesToShip.Identity)').Contains('latest'))" />
+    </ItemGroup>
+    <Copy SourceFiles="@(PackagesToShip)"
```

Comment:
You don't need to rely on `$(ShippedNuGetPackageGlobPath)`.  You can just set `@(ShippedNuGetPackage)` directly, and it will work fine.

See https://github.com/dotnet/buildtools/blob/8c000f1c9d79d4d3ba711585a7212619dd86749d/src/Microsoft.DotNet.Build.Tasks/PackageFiles/VersionTools.targets#L20-L25

If you set the globpath, build-tools will use it to populate `@(ShippedNuGetPackage)`, and then pass that item into the `UpdatePublishedVersions` task.  I'm saying, skip the glob path all together, as it doesn't bring us any value.  Instead just directly set `@(ShippedNuGetPackage)` yourself, and that will get passed into the `UpdatePublishedVersions` task

/cc @dagood